opendg/timedelta.py

```diff
@@ -0,0 +1,117 @@
+from typing import Optional, Union
+
+# from opendg.typing import TimeDeltaTG
+
+
+class TimeDeltaTG:
+    r"""TimeGranularity class to represent time granularity in dynamic graph."""
+
+    def __init__(self, interval_type: str, duration: Optional[int] = 1):
```

Comment:
```suggestion
    def __init__(self, interval_type: str, duration: int = 1):
```

Duration is not `Optional`, because it cannot be `None`.